### DISCUSSION TITLE
[Question] Can we use Flutter inspector with Flet?

### DISCUSSION THREAD

**Discussion by torablien** [flet-dev/flet#3110] [Q&A]
### Question

Are we able to use Flutter inspector with Flet? Occasionally, I want to debug layout issues but it doesn't seem easily possible in Flet (short of putting containers with red borders around all the widgets). Any suggestions?

I see some mentions of `debug` mode but I'm not sure how to set it. I came across this from 2 years ago which might be related: https://github.com/flet-dev/flet/issues/106

Thanks!

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by dougcooper** [comment]
did you ever figure this out?
